### ISSUE TITLE
Capturing relativism for ACMG-based assertions

### ISSUE THREAD

**Issue by rrfreimuth** [tnavatar/clingen-data-model#131]
I don't think I expressed my questions well on the call today.  Hopefully this helps...

Given a variant at genomic position 123:
Genomic ref allele = A
Alt allele known at this site = G
Transcript 1 contains A, predicted AA sequence = Pro
Transcript 2 contains G, predicted AA sequence = Ala

Phenotypically, the protein containing Pro is known to have reduced activity, so the A allele is considered damaging.  The protein containing Ala, however, is considered to produce “normal” activity.

Questions:
1.  Since the genomic ref (build) contains A, is A considered the ref allele?
2.  If A (Pro) is “reference”, would the G allele considered damaging (computationally)?
3.  Do we need a way for a curator to reverse this situation?  E.g., the G allele (Ala) is considered “normal” and the A allele (Pro) is asserted to be pathogenic

Since there is potentially a difference between the “reference” allele (relative to the genome build) and “reference” functional activity (which is assessed by curators), it seems we need the ability to explicitly capture this information rather than inferring it from the genomic data.

For example, we could make the following statements:
• G>A is pathogenic (missense) if transcript 2 is used for baseline functional activity.  This would be the point of view for curators.
• A>G is NOT pathogenic if transcript 1 is used for baseline activity (in fact, it would result in increased activity, not sure how this would be represented by ACMG).  This might (?) be the point of view for computational algorithms that use the genomic ref sequence as “reference”.

To add further complexity, let’s pretend that Transcript 3 is formed through alternative splicing (or utilizes an alternative transcription initiation site) and does not contain genomic position 123.  In that case, either allele would be considered to be silent and NOT pathogenic.

Bottom line:  I think we need to capture enough information about the decisions made using the ACMG criteria so that there is no ambiguity about where the assertions came from.  We may be able to infer some of the info via Simple/Canonical Allele, but it may not always be unambiguous (this is the question I was trying to ask today, so forgive me if the model handles it and I wasted time on the call).

**Comment by srynobio** [comment]
Some thought on your questions.  Feedback, dispute and clarification welcome.

Let's also assume were talking about the same build i.e. b37

> Since the genomic ref (build) contains A, is A considered the ref allele?

'A' would be considered the genomic reference, and any annotation made based on a given set of transcripts would come from the same build (else be inaccurate).

> If A (Pro) is “reference”, would the G allele considered damaging (computationally)?

The "damage" a genomic change has can only be assessed based on one (or more) give transcripts, and the resulting protein change each may have.

> Do we need a way for a curator to reverse this situation? E.g., the G allele (Ala) is considered “normal” and the A allele (Pro) is asserted to be pathogenic

I'm unsure what you mean by reverse.  If this situation arose where (in the same build) a A>G change was noted as pathogenic and another SimpleAllele G>A was also noted as pathogenic it would denote a data/interpretation issue (maybe annotated with two different builds).

> For example, we could make the following statements:
> • G>A is pathogenic (missense) if transcript 2 is used for baseline functional activity. This would be the point of view for curators.
> • A>G is NOT pathogenic if transcript 1 is used for baseline activity (in fact, it would result in increased activity, not sure how this would be represented by ACMG). This might (?) be the point of view for computational algorithms that use the genomic ref sequence as “reference

Given the breakdown:

```
Transcript 1 -> G>A
Transcript 2 -> A>G
```

There seem to be an issue here because you describing the same genomic variant in the context of two different transcripts, but with a 'G' in one transcript and an 'A' in another at the same genomic location is not possible (if you're describing the same variant).

> To add further complexity, let’s pretend that Transcript 3 is formed through alternative splicing (or utilizes an alternative transcription 
> initiation site) and does not contain genomic position 123. In that case, either allele would be considered to be silent and NOT pathogenic.

I would agree that any new transcripts found which do not include position 123, would not be pathogenic, if it's discovered that transcript 3 plays a functional role.  I don't think it would negate the role transcript 1 & 2 has (i.e. case dependent).

**Comment by larrybabb** [comment]
If anyone would like to take on the challenge of creating 'real' example json files that highlight the scenarios above, it would be very helpful in clarifying the various issues and identifying changes that are needed in the model and/or documentation.
